Library/Source/NativeEngine.cpp

```diff
@@ -398,7 +402,7 @@ namespace babylon
 
     void NativeEngine::Impl::Initialize(Napi::Env& env)
     {
-        EngineDefiner::Define(env, this);
+        //EngineDefiner::Define(env, this); todo
```

Comment:
I think this _has_ to be called; if it isn't, Napi won't know what a NativeEngine is when Babylon asks for one. What was the error this was causing?

Reply:
yep, I put a comment on that line. template madness here. I tried to fix without success